src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,287 @@ paths:
         500:
           description: "Internal Server Error"
 
+  /alert-subscriptions:
+    post:
+      operationId: "create_and_subscribe_to_alert"
+      tags:
+        - "Alert-Subscriptions"
+      summary: Creates an alert and subscribes emailIds for a particular assetID
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/x-gw-ims-org-id'
+        - $ref: '#/parameters/x-sandbox-name'
+        - $ref: '#/parameters/x-sandbox-id'
+        - $ref: '#/parameters/x-service-token'
+        - $ref: '#/parameters/x-api-key'
+        - $ref: '#/parameters/x-request-id'
+        - $ref: '#/parameters/User-Agent'
```

Comment:
Where is the `Authorization` header? This is a required header for all Platform endpoints.

The top headings for all endpoints should be (in the following order):
- `Authorization`
- `x-gw-ims-org-id`
- `x-api-key`
- `x-sandbox-name`

All other headers/params can be listed after, grouped by type (`header`/`path`/`query`) with required params listed before optional params.

Reply:
I am not sure what you mean by "...grouped by type (header/path/query) ...". I assume you mean their descending order of listing?